src/pages/welcome/Welcome.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, H2, UL } from '@blueprintjs/core';
+import { Links } from 'src/commons/utils/Constants';
+
+export type WelcomeProps = StateProps;
+
+export type StateProps = {
+  name?: string;
+};
+
+const Welcome: React.FC<WelcomeProps> = props => {
+  return (
+    <div className="fullpage">
+      <Card className="fullpage-content">
+        <div style={{ marginTop: '20px', display: 'flex', justifyContent: 'center' }}>
+          <div>
+            <H2>Welcome to Source Academy @ NUS</H2>
+            <div>
+              You have logged in as <strong>{props.name}</strong>. Source Academy @ NUS does not
```

Comment:
Maybe we should move this "Source Academy @ NUS" title to a configuration env var